packages/minimal/src/managers/rmf-health-state-manager.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  ItemSummary,
+  RobotSummary,
+  SpoiltDispenser,
+  SpoiltDoor,
+  SpoiltLift,
+} from 'react-components';
+
+export interface HealthStatus {
+  door: ItemSummary<SpoiltDoor>;
+  lift: ItemSummary<SpoiltLift>;
+  dispenser: ItemSummary<SpoiltDispenser>;
+  robot: RobotSummary;
+}
+
+// TODO - fill up health state manager logic once backend is up
+export default class RmfHealthStateManager {
```

Comment:
Duplicated code, I don't think this is used, but if it needs to be, it should probably be moved to `react-components`.